ext/spl/tests/ArrayObject_overloaded_object_incompatible.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 Objects with overloaded get_properties are incompatible with ArrayObject
+--XFAIL--
+SplFixedArray has migrated to get_properties_for; find new test case
```

Comment:
I guess you could add a stub class to ext/zend_test and move this test there for unit testing?